hardhat.config.ts

```diff
@@ -158,24 +158,31 @@ const config: HardhatUserConfig = {
   deploymentManager: {
     relationConfigMap,
     networks: {
-      fuji: fujiRelationConfigMap,
-      kovan: kovanRelationConfigMap,
+      fuji: {
+        usdc: fujiUSDCRelationConfigMap,
+      },
+      kovan: {
+        usdc: kovanUSDCRelationConfigMap,
+      }
     },
   },
 
   scenario: {
     bases: [
       {
         name: 'development',
+        deployment: 'usdc'
       },
       {
         name: 'kovan',
+        deployment: 'usdc',
```

Comment:
Can we get rid of the network params now that we have the network name for the deployment?

Reply:
we'll still need it for scenarios, right?

scenarios will iterate over `bases`; for each base it finds, it will generate an hre using the `name` field, and then look for a `roots.json` file in `${name}/${deployment}/roots.json`.

seems like we should rename `name` to `network`

Reply:
Well but the network is configured as part of the deployment now, that was the whole point of including it in the deployment info (encoded in path currently, alternatively was `network.json`)? So `base` has a name so that we can refer to it from `--bases`, and it uses a certain deployment, but should be able to pull chain args from the deployment network, right?

Reply:
So if you've got two deployments:

`/deployments/kovan/usdc`
`/deployments/fuji/usdc`

and your bases are:

```
[
  {
    deployment: 'usdc'
  },
  {
    deployment: 'usdc'
  }
]
```

how would you specify that you only wanted to run the scenarios for `fuji/usdc`?

```
yarn scenario --bases usdc
```

or are you saying we should have the deployment name be `fuji/usdc`?

Reply:
I guess it really needs to be `fuji/usdc` and `kovan/usdc`; right now, we're limited to running scenarios on a single deployment per network

Reply:
It's the base name not the deployment name! Technically both deployments are named `usdc`, the bases are named whatever you name them in the hardhat config